README.md

```diff
@@ -445,6 +440,31 @@ maintains a list of these curators.
 e49458a13f8a000b5959d03f8f7b6fa397b578643940ba50d3470c201d333429 added as a curator to allocation fb84185dae620bbba8386286726f1efcd20d2516bcf1a448215434d87be3b30d
 ```
 
+## Remove curator
+A curator can be removed for an allocation by using the `removecurator` command , 
```

Comment:
A curator can be removed ~for~ `from` an allocation by using the

or

An allocation's curator can be removed using the